driver/src/test/java/org/kaazing/k3po/driver/internal/RobotIT.java

```diff
@@ -468,6 +471,129 @@ public void shouldEcho() throws Exception {
     }
 
     @Test
```

Comment:
I just saw that some tcp tests are there in RobotIT and I added them. If you want I can create UdpIT for these two tests using @TestSpecification.